packages/react-app/src/components/modals/reclaim-funds-modal.tsx

```diff
@@ -107,7 +107,9 @@ export default function ReclaimFundsModal({ questData, bounty, onClose = noop }:
             onClick={() => setOpened(true)}
             icon={<IconCoin />}
             label="Reclaim funds"
-            disabled={!bounty.parsedAmount}
+            disabled={
```

Comment:
You dont need it if you handle bounty on quest side because the component will not be loaded (remember with the)
![image](https://user-images.githubusercontent.com/13890618/165757919-35cc8972-f807-4062-837d-e3b8f28d508b.png)